src/main/java/core/basesyntax/model/ma/Person.java

```diff
@@ -1,6 +1,7 @@
 package core.basesyntax.model.ma;
 
 public class Person {
+    private Long id;
```

Comment:
should we add id field to other base classes as well?

Reply:
in the next PR I will add it. This is required for the test cases